chart/cas-cif/values-test.yaml

```diff
@@ -5,12 +5,18 @@ app:
 hostName: test.cif.gov.bc.ca
 
 db:
-  preUpgradeCommand: |
+  postUpgrade: true
+  upgradeCommand: |
```

Comment:
This api doesn't allow us to do both pre- and post- upgrade, I think we should keep them separate just in case, to avoid confusion